docs/docs/api/LayoutAnimations/ExitAnimations.md

```diff
@@ -6,7 +6,8 @@ sidebar_label: Exiting Animations
 
 In React Native during unmounting of components from the hierarchy of views, it just disappears in the next frame. However you can beautify this process using `Exiting Animations`. Reanimated make a pretty animation of disappearing of component for you.
 #### How it is possible? 
-Reanimated listen on changes in tree of views and if detect that some of component should disappear in next frame, It replaces this process with exiting animation. It is easy and fast. You can use predefined animations - examples below or you can define your own custom animation.
+Reanimated listen on changes in tree of views and if detect that some of component should disappear in next frame, It replaces this process with exiting animation. It is easy and fast. You can use predefined animations - examples below or you can define your own custom animation. If you want to create more complex exiting animation you can use [Keyframes](keyframeAnimations).
```

Comment:
Why only "exiting animation"? Doesn't it support entering animation too?

Reply:
Yes, and in `EntryAnimations.md` I wrote "entering animation". However, to make it more clear, I'm gonna change it to "complex animation" in both cases.